cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc

```diff
@@ -9,23 +9,23 @@ The syntax is:
 | Command | Description
 
 | [source, cypher]
-GRANT database-privilege ON DATABASE[S] {dbname \| *} TO role[, ...]
+GRANT database-privilege ON DATABASE[S] {* \| name} TO role[, ...]
 | Grant a privilege to one or multiple roles
 
 | [source, cypher]
-DENY database-privilege ON DATABASE[S] {dbname \| *} TO role[, ...]
+DENY database-privilege ON DATABASE[S] {* \| name} TO role[, ...]
 | Deny a privilege to one or multiple roles
 
 | [source, cypher]
-REVOKE GRANT database-privilege ON DATABASE[S] {dbname \| *} FROM role[, ...]
+REVOKE GRANT database-privilege ON DATABASE[S] {* \| name} FROM role[, ...]
 | Revoke a granted privilege from one or multiple roles
 
 | [source, cypher]
-REVOKE DENY database-privilege ON DATABASE[S] {dbname \| *} FROM role[, ...]
+REVOKE DENY database-privilege ON DATABASE[S] {* \| name} FROM role[, ...]
 | Revoke a denied privilege from one or multiple roles
 
 | [source, cypher]
-REVOKE database-privilege ON DATABASE[S] {dbname \| *} FROM role[, ...]
+REVOKE database-privilege ON DATABASE[S] {* \| name} FROM role[, ...]
 | Revoke a granted or denied privilege from one or multiple roles
 |===
 
```

Comment:
Since graph privileges came first and are before the database privileges if you read through the whole chapter (it was differently structured back when we started so it was more logical) I think we added it to show differences. I think we have added a similar link from the graph privilege section pointing here as well at some point.

I guess I could remove the `and` (and make sure the other note don't have two as well)

Reply:
I think these notes might have been removed in 4.1